docs/one-click-integrations.md

```diff
@@ -54,6 +54,7 @@ The Pulse GitHub integration supports two strategies to detect and measure deplo
     -   The set of changes that belong to a deployment is the list of commits between the tag of that deployment and the previous tag.
 
         Because of this, Pulse discards the first SemVer tag in the repository since there is no previous tag to compare with.
+        Pulse also discards any tag that does not have a common ancestor (commit) with its previous tag, since it cannot obtain the changes between them.
```

Comment:
Suggestion to condense the text a bit, as the description of how Pulse works with Git tags is growing quite large:

```suggestion
    -   The set of changes that belong to a deployment is the list of commits between the tag of that deployment and the previous tag. Because of this, Pulse discards:
        
        -   The first SemVer tag in the repository since there is no previous tag to compare with.
        -   Any tag that does not have a common ancestor (commit) with its previous tag, since Pulse cannot obtain the changes between them.
```